lib/readline.js

```diff
@@ -409,9 +407,10 @@ Interface.prototype._normalWrite = function(b) {
   if (b === undefined) {
     return;
   }
+  const now = Date.now();
   var string = this._decoder.write(b);
   if (this._sawReturnAt &&
-      now() - this._sawReturnAt <= this.crlfDelay) {
+      now - this._sawReturnAt <= this.crlfDelay) {
```

Comment:
On second thought, `now` is a costant, so isn't this always 0?

Reply:
It's constant within the scope of the function but the case it's accounting for is when `_normalWrite` is called twice in a row, once with `\n` and then with `\r`.

Reply:
Gotcha.

Reply:
That said, this did make me rethink caching the value like I did. It doesn't make any sense and it's an unnecessary call in many situations. I'll fix that up.

Reply:
Also it's definitely easier to grok now.